store/src/java/com/zimbra/cs/service/ZimbraAuthProvider.java

```diff
@@ -125,8 +126,8 @@ protected AuthToken authToken(Element soapCtxt, Map engineCtxt, TokenType tokenT
             if (!StringUtil.isNullOrEmpty(jwt)) {
                 try {
                     Claims body = Auth.validateJWT(jwt);
-                    String encodedAuthToken = (String) body.get(Constants.AUTH_DATA_CLAIM);
-                    at = genAuthToken(encodedAuthToken);
+                    Account acct = Provisioning.getInstance().getAccountById(body.getSubject());
```

Comment:
How will this code work if acct is null at this point (perhaps the account has just been deleted?)

Reply:
added null check for account.